### PR TITLE
Life Events section on front page + Section component

### PR DESCRIPTION
**Pull request by DavidBachmann**
Implement Life Events for the front page. It uses dummy data so we can display something for the demo tomorrow.

image

Implement a Section component that deals with section spacing and knows how to deal with background bleeding like this:
image

### CODE REVIEW

File: apps/web/components/Section/Section.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  return (
+    <Box
+      as={as}
+      paddingY={paddingY}
+      paddingTop={paddingTop}
+      paddingBottom={paddingBottom}
+      background={background}
+      style={backgroundBleed && generateBackgroundBleed(backgroundBleed)}
```
Comment (jeremybarbet) [inline apps/web/components/Section/Section.tsx:55]:
```suggestion
      style={generateBackgroundBleed}
```

File: apps/web/components/Section/Section.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  paddingTop,
+  paddingBottom,
+  background,
+  backgroundBleed,
+}) => {
+  const { width } = useWindowSize()
+
+  const generateBackgroundBleed = (backgroundBleed: BackgroundBleed) => {
+    //Background bleed is not available on mobile.
+    if (width < theme.breakpoints.md) {
```
Comment (jeremybarbet) [inline apps/web/components/Section/Section.tsx:35]: I'm surprised, shouldn't you get an error for using `backgroundBleed` as a prop and as a function argument?

```suggestion
    if (!backgroundBleed  || width < theme.breakpoints.md) {
```

**Review by jeremybarbet — APPROVED** [review]
LGTM

File: apps/web/components/Section/Section.tsx (hunk 1 above)
Comment (DavidBachmann) [reply apps/web/components/Section/Section.tsx]: You mean `generateBackgroundBleed()`. Implemented.

File: apps/web/components/Section/Section.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  children,
+  paddingY,
+  paddingTop,
+  paddingBottom,
+  background,
+  backgroundBleed,
+}) => {
+  const { width } = useWindowSize()
+
+  const generateBackgroundBleed = (backgroundBleed: BackgroundBleed) => {
```
[the remark this answers is not in the document]
Comment (DavidBachmann) [reply apps/web/components/Section/Section.tsx]: Implemented

File: apps/web/components/Section/Section.tsx (hunk 2 above)
Comment (DavidBachmann) [reply apps/web/components/Section/Section.tsx]: Implemented. Thanks.

**Pull request merged**